### PR TITLE
Fix UFS RPC metrics aggregation

### LINKED ISSUE #9628: Metrics JSON endpoint not reporting correct UFS metrics
**Alluxio Version:**
1.8.2

**Describe the bug**
UFS metrics are not reported correctly. When creating two different mount points on the same HDFS cluster, operations on the second mount point are incorrectly reported to the first mount point.

**To Reproduce**
Mount two different directories from an HDFS cluster into the same Alluxio cluster. RunTests on the second mount directory. Check the /metrics/json/ endpoint and see the value of getBlockSizeBytes().

**Expected behavior**
Operations like getBlockSizeBytes should be reported based on which mount point they were run on instead of aggregating in the first mount point.

**Urgency**
Medium

**Additional context**
Was not able to reproduce in 2.x

### PR DESCRIPTION
**Pull request by yuzhu**
Originally, the metrics are aggregated on UFS paths. But in scenario where two paths in the same UFS are mounted to different Alluxio mount points, since there is only one UFS client for the mount points, all metrics are aggregated to the first mount point even when operations are on the second mount point, which is confusing.
This PR aggregates UFS RPC metrics per UFS, instead of per UFS path.

fixes #9628

### CODE REVIEW

**Comment by yuzhu** [comment]
@chcn ported #9846 to target master branch. PTAL

**Comment by yuzhu** [comment]
@ZacBlanco @rastogiasr what is the verdict on this? to master but not to branch-2.1?

**Comment by yuzhu** [comment]
@chcn PTAL

File: core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java [hunk 1]
```diff
@@ -961,22 +963,26 @@ public boolean isSeekable() {
     return mUnderFileSystem.isSeekable();
   }
 
+  private String getUfsTag() {
+    AlluxioURI uri = new AlluxioURI(mPath);
+    uri = new AlluxioURI(uri.getScheme(), uri.getAuthority(), uri.getPath());
```
Comment (chcn) [inline core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java]: what's the purpose of this?

Comment (yuzhu) [reply core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java]: emm this is from your old PR...

Comment (yuzhu) [reply core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java]: #9846

**Pull request closed without merging**